### PR TITLE
fix(alerts): Allow mep-rollout-flag to enable metric dataset

### PR DESCRIPTION
**Pull request by scttcper**
related #37435

### CODE REVIEW

File: static/app/views/alerts/rules/metric/ruleConditionsForm.tsx [hunk 1]
```diff
@@ -436,26 +436,29 @@ class RuleConditionsForm extends PureComponent<Props, State> {
...
-            </AlertContainer>
-          )}
+        {showMEPAlertBanner && hasMetricDataset && (
+          <AlertContainer>
+            <Alert type="info" showIcon>
+              {tct(
+                'Filtering by these conditions automatically switch you to indexed events. [link:Learn more].',
+                {
+                  link: (
+                    <ExternalLink href="https://docs.sentry.io/product/sentry-basics/search/" />
```
Comment (wmak) [inline static/app/views/alerts/rules/metric/ruleConditionsForm.tsx:455]: unrelated; is this just a placeholder link for now? 🤔

**Review by wmak — APPROVED** [review]
lgtm

**Review by wedamija — APPROVED** [review]

File: static/app/views/alerts/rules/metric/ruleConditionsForm.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/views/alerts/rules/metric/ruleConditionsForm.tsx:455]: hmm the copy isn't great either

Comment (wmak) [reply static/app/views/alerts/rules/metric/ruleConditionsForm.tsx:455]: @scttcper The copy we use in dashboards is
> Your selection is only applicable to stored event data. We've automatically adjusted your results.

no docs mention too

**Pull request merged**